stb_vorbis.c

```diff
@@ -92,14 +92,18 @@ typedef unsigned short uint16;
 typedef   signed short  int16;
 typedef unsigned int   uint32;
 typedef   signed int    int32;
-#if defined(__x86_64) || defined(__x86_64__) || defined(__amd64) || defined(_M_X64) \
+#if defined(__x86_64) || defined(__x86_64__) || defined(__amd64) \
```

Comment:
All these three already define `_LP64` or `__LP64__` or both, they are nonsense here.

Reply:
Any checks other than `_LP64` or `__LP64__` can and should go away here.

Reply:
Yes

Reply:
Should be fine now.